CondFormats/RunInfo/src/FillInfo.cc

```diff
@@ -208,6 +208,13 @@ std::string const & FillInfo::injectionScheme() const {
   return m_injectionScheme;
 }
 
+//returns a boolean, true if the injection scheme has a leading 25ns
+//TODO: parse the circulating bunch configuration, instead of the string.
+bool FillInfo::is25nsBunchSpacing() const {
+  std::string prefix( "25ns" );
```

Comment:
@davidlange6 true, but anyhow goes out of scope after the function call...
Implementing the fix.